content/en/docs/features/control-plane/entity-tag.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'Entity-tag (ETAG)'
+description: 'What is ETAG?'
+docsOthersDisplay: true
+date: '2021-05-13'
+categories: [features]
+keywords: [twin, twin, cache, history]
+weight: 31
+---
+
+An entity-tag (ETAG) serves as a resource-local identifier to distinguish between different representations of the same resource that change over time. It is generated by the server (device) providing the resource and can be created using various methods such as a version, checksum, hash, or timestamp. When an endpoint (client) receives an entity-tag, it should treat it as an opaque value without making any assumptions about its content or structure. In CoAP, the ETag is part of the [CoAP options](https://datatracker.ietf.org/doc/html/rfc7252#section-5.4) and is used to detect changes in resources. It optimizes resource serialization on the device side and resource deserialization on the client side.
```

Comment:
> It is used optimizes the client processing time of the response from the device in case no change was detected - the ETAG wasn't changed from the last request.

But it also optimize at device side because the representation is not serialized.